dm/master/bootstrap.go

```diff
@@ -369,6 +372,19 @@ outerLoop:
 				}
 			}
 		}
+		sort.Slice(cfgLists, func(i, j int) bool {
+			return cfgLists[i].SourceID < cfgLists[j].SourceID
+		})
+		var mergedCfg config.TaskConfig
+		err2 := mergedCfg.FromSubTaskConfigs(cfgLists...)
```

Comment:
seems `FromSubTaskConfigs` doesn't have a fail return

Reply:
👍 removed return value in https://github.com/pingcap/dm/pull/911/commits/7eb2f69decff28977614ca70da29f165d69d70db.